Dockerfile

```diff
@@ -44,4 +42,4 @@ COPY --from=build /go/bin/goshimmer /run/goshimmer
 # Copy the default config
 COPY config.default.json /config.json
 
-ENTRYPOINT ["/run/goshimmer", "--config-dir=/", "--database.directory=/mainnetdb"]
+ENTRYPOINT ["/run/goshimmer", "--config-dir=/", "--database.directory=/tmp/mainnetdb"]
```

Comment:
The base image does not have `/bin/sh` to run `mkdir /mainnetdb`. But as a workaround I used `WORKDIR` to create the directory now.

Reply:
This does, however, only work if there is a mounted volume which has `rw` rights. Inside the container (for the network) this does not work. No permission to write into this folder.

I'll leave it with `/tmp/mainnetdb`. 

Btw which tools are you referring to?

Reply:
@nuriel77 what do you think?
IIRC your playbook depends on the `mainnetdb` path. Is it ok to move it to `/tmp/mainnetdb` or, if necessary, do you have any proposals on how to keep it under `/mainnetdb` that are compatible with running the container without any mounts?

Reply:
@Wollac Why does anything have to be hard-coded for the entry point?
For me doesn't matter much because I can also override the default entry point.
Nevertheless, I think it is best practice to avoid hard coding.

Why not use:
```
ENTRYPOINT ["/run/goshimmer", "--config-dir=/", "$@"]
```
Or even:
```
ENTRYPOINT ["/run/goshimmer", "--config-dir=/", "$@"]
```
And set the defaults inside of goshimmer? (i.e. config dir=/ and db dir to /tmp/etc...)?